src/pages/Login/index.js

```diff
@@ -0,0 +1,70 @@
+import React, { useContext, useState } from 'react'
+import * as Styled from './styles'
+import AppContext from '../../context/AppContext';
+import { Toast, MessageTypes, ModalMessage } from '../../utils/Message';
+import LabelEnv from '../../components/LabelEnv'
+import { login } from '../../services/login/Auth';
+import { Form, FormGroup, FormControl } from 'react-bootstrap'
+import { Redirect } from "react-router-dom";
+
+
+function Login() {
+    const { loading } = useContext(AppContext);
+    const [email, setEmail] = useState('');
+    const [password, setPassword] = useState('');
+
+    function validateForm() {
+        return email.length > 0 && password.length > 0;
```

Comment:
O zero já representa falso 
```suggestion
        return email.length && password.length;
```

Reply:
alterado